scripts/embargo_registrations.py

```diff
@@ -24,12 +24,12 @@
 
 
 def main(dry_run=True):
-    pending_embargoes = Embargo.objects.filter(state=Embargo.UNAPPROVED)
+    pending_embargoes = Embargo.objects.select_for_update().filter(state=Embargo.UNAPPROVED)
     for embargo in pending_embargoes:
         if should_be_embargoed(embargo):
             if dry_run:
                 logger.warn('Dry run mode')
-            parent_registration = Registration.objects.get(embargo=embargo)
+            parent_registration = Registration.objects.select_for_update().get(embargo=embargo)
```

Comment:
Drive-by: Are you sure that parent_registration is modified in this script? `select_for_update` should only be used when you are making an update.